tests/supply_chain/test_supply_chain.py

```diff
@@ -1287,8 +1288,8 @@ def test_distribution_unit_dispatch_order(self) -> None:
         distribution_unit = supplier_3.distribution
         product_unit = supplier_3.products[SKU3_ID]
 
-        vehicle_1: VehicleUnit = distribution_unit.vehicles["train"][0]
-        vehicle_2: VehicleUnit = distribution_unit.vehicles["train"][1]
+        vehicle_1: Vehicle = distribution_unit.vehicles["train"][0]
```

Comment:
Cannot find the definition of `Vehicle`

Reply:
left unit tests to be TODO items in this commit